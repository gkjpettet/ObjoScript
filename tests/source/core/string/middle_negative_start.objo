System.print("hello".middle(-1)) # Expect error.