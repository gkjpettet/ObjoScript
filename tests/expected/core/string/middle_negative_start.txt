The `start` argument must be a positive integer.